src/Spectre.libException/EmptyOpenCvDatasetException.h

```diff
@@ -0,0 +1,41 @@
+﻿/*
```

Comment:
This file should not be in `libException`. There are general-purpose exceptions, which span through the whole solution. This exception is specific and narrowed down to a single use-case, so it should be at this place - in `libClassifier` project, just besides `OpenCvDataset.cpp` \ `OpenCvDataset.h`.